FirebaseUI/Auth/README.md

```diff
@@ -37,8 +37,8 @@ pod 'FirebaseUI/Auth'
 To use FirebaseUI to authenticate users you first need to configure each provider you want to use in
 their own developer app settings. Please read the *Before you begin* section of the Firebase
 Auth guides at the following links:
-[Email and password](https://firebase.google.com/docs/auth/web/password-auth#before_you_begin)
-[Google](https://firebase.google.com/docs/auth/ios/google-signin#before_you_begin)
+[Email and password](https://firebase.google.com/docs/auth/ios/password-auth#before_you_begin),
```

Comment:
I think this was supposed to be a bulleted list:
  - [Email and password](https://firebase.google.com/docs/auth/ios/password-auth#before_you_begin)
  - [Google](https://firebase.google.com/docs/auth/ios/google-signin#before_you_begin)
  - [Facebook](https://firebase.google.com/docs/auth/ios/facebook-login#before_you_begin)